ksqldb-functional-tests/src/test/resources/query-validation-tests/bytes.json

```diff
@@ -1,5 +1,5 @@
 {
-  "comments": ["tests for decimal functionality"],
+  "comments": ["tests for bytes functionality"],
```

Comment:
Nice fix :)

Reply:
what would the ksql team do without me? :')